tests/ops/test_qubit_ops.py

```diff
@@ -499,10 +499,7 @@ def test_adjoint_unitaries(self, op, tol):
 
     @pytest.mark.parametrize(
         "op_builder",
-        [
-            lambda: qml.QFT(wires=[1, 2, 3]),
-            lambda: qml.QubitCarry(wires=[0, 1, 2, 3]),
-        ],
+        [lambda: qml.QFT(wires=[1, 2, 3]), lambda: qml.QubitCarry(wires=[0, 1, 2, 3]),],
```

Comment:
Running black has led to a number of changes to the file this time around :confused: